renderer/components/Pay/Pay.js

```diff
@@ -238,7 +238,7 @@ class Pay extends React.Component {
     }
 
     // Change the transaction filter to ALL transactions
-    changeFilter('SENT_ACTIVITY')
+    addFilter('SENT_ACTIVITY')
```

Comment:
After sending a new payment, the payment will be in pending state - so will not be included in the `SENT_ACTIVITY` filter. This should probably also add `PENDING_ACTIVITY`